content/en/blog/2020/docker-rate-limit/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Handling Docker Hub rate limiting
+description: How to ensure your clusters are not impacted by Docker Hub rate limiting.
+publishdate: 2020-12-05
+attribution: John Howard (Google)
+keywords: [docker]
+target_release: 1.8
+---
+
+Since November 20th, 2020, Docker Hub has introduced [rate limits](https://www.docker.com/increase-rate-limits) on image pulls.
+
+Because Istio uses [Docker Hub](https://hub.docker.com/u/istio) as the default registry, usage on a large cluster may lead
+to pods failing to startup due to exceeding rate limits. This can be especially problematic for Istio, as there is typically
+the Istio sidecar image alongside most pods in the cluster.
+
+# Mitigations
+
+Istio allows providing a custom docker registry which can be used so that the images can be fetched from your private registry. This can be configured by passing `--set hub=<some-custom-registry>` at installation time.
```

Comment:
```suggestion
Istio allows providing a custom docker registry which can be used so that the images are fetched from your private registry. This can be configured by passing `--set hub=<some-custom-registry>` at installation time.
```